seqr/views/apis/dataset_api_2_3_tests.py

```diff
@@ -226,9 +233,32 @@ def test_receive_alignment_table_handler(self):
             'uploadedFileId': mock.ANY,
             'errors': [],
             'info': ['Parsed 2 rows from samples.csv', 'No change detected for 1 individuals'],
-            'updatesByIndividualGuid': {'I000003_na19679': 'gs://readviz/NA19679.bam'},
+            'updatesByIndividualGuid': {
+                'I000003_na19679': 'gs://readviz/NA19679.bam',
+            },
         })
 
+        # Test with tsv data formats
+        f = SimpleUploadedFile('samples.tsv', "NA19675_1\t/readviz\xe2/NA19675.cram\nNA19679\tgs://readviz\xe3/NA19679.bam".encode('utf-8'))
+        response = self.client.post(url, data={'f': f})
+        self.assertEqual(response.status_code, 200)
+        self.assertListEqual(response.json()['info'], ['Parsed 2 rows from samples.tsv'])
+
+        # Test with xlsx data formats
+        wb = xl.Workbook()
+        ws = wb[wb.sheetnames[0]]
+        ws['A1'], ws['B1'] = 'NA19675_1', '/readviz\xe2/NA19675.cram'
+        ws['A2'], ws['B2'] = 'NA19679', 'gs://readviz\xe3/NA19679.bam'
+        file = tempfile.TemporaryFile()
+        wb.save(file)
+        file.seek(0)
+        data = file.read()
+        file.close()
+        f = SimpleUploadedFile('samples.xlsx', data)
+        response = self.client.post(url, data={'f': f})
+        self.assertEqual(response.status_code, 200)
+        self.assertListEqual(response.json()['info'], ['Parsed 2 rows from samples.xlsx'])
+
```

Comment:
Added tests for uploading `.tsv` and `.xlsx` files.

Reply:
I don't think this is a good place for adding a test for xls file parsing. I think you should do that in the explicit tests for `parse_file` in `file_utils_tests` (which already exist). You can keep the tsv if you want because its relatively simple, although its not necessary

Reply:
Remove these tests. Adding them to the tests for file_utils.py.

Reply:
yes I agree